gcp_variant_transforms/libs/bigquery_vcf_schema_converter_test.py

```diff
@@ -50,6 +52,14 @@ def modify_bigquery_schema(self, schema, info_keys):
 class GenerateSchemaFromHeaderFieldsTest(unittest.TestCase):
   """Test cases for the ``generate_schema_from_header_fields`` function."""
 
+  def _validation_hook(self, expected_fields, actual_schema):
```

Comment:
nit: why not just name this `_validate_schema` , as it's more obvious what it's validating (unless you're trying to validate more stuff, but the comments say otherwise).